.eslintignore

```diff
@@ -7,3 +7,4 @@ es/
 lib/
 _site/
 dist/
+types/
```

Comment:
why ?

Reply:
typescript not support in eslint config currently